src/Microsoft.Data.SqlClient/tests/tools/TDS/TDS.EndPoint/TDS.EndPoint.csproj

```
@@ -41,8 +41,8 @@
     <Compile Include="TDSServerParser.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
-    <Compile Include="$(CommonPath)\Interop\Windows\Interop.Libraries.cs">
-      <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
+    <Compile Include="$(CommonSourceRoot)\Interop\Windows\Interop.Libraries.cs">
+      <Link>Interop\Windows\Interop.Libraries.cs</Link>
```

Nit: Does this need to change and be different from the other projects?
```suggestion
      <Link>Common\Interop\Windows\Interop.Libraries.cs</Link>
```

---

I guess the way I look at it, the interop code isn't particularly "common". And, generally speaking I think we should be following a pattern where the file paths match their namespaces. So to me, the common folder seems redundant.